### PR TITLE
chore(functions/firebase): fix samples + tests not running

### PR DESCRIPTION
**Pull request by ace-n**
Related to cl/330629293

### CODE REVIEW

File: functions/firebase/index.js [hunk 1]
```diff
@@ -89,16 +89,23 @@ const firestore = new Firestore({
...
   const affectedDoc = firestore.doc(resource.split('/documents/')[1]);
 
   const curValue = event.value.fields.original.stringValue;
   const newValue = curValue.toUpperCase();
-  console.log(`Replacing value: ${curValue} --> ${newValue}`);
 
-  return affectedDoc.set({
-    original: newValue,
-  });
+  if (curValue != newValue) {
```
Comment (grant) [inline functions/firebase/index.js:98]: !==

**Review by grant — APPROVED** [review]
Looks okay, although logic doesn't look like other samples exactly.
http://devrel/snippets/functions_firebase_reactive

**Comment by ace-n** [comment]
The other samples are being updated a bit too FYI - see b/168072108

File: functions/firebase/index.js (hunk 1 above)
Comment (ace-n) [reply functions/firebase/index.js]: Done.

**Pull request merged**